src/main/webapp/release_sdks.html

```diff
@@ -26,7 +26,7 @@ <h1 id="header"></h1>
                 </div>
             </div>
         </div>
-
+        
```

Comment:
Remove the spaces here

Reply:
there  is space after every div so that will  be incosistent if I remove